databases/week1/homework.sql

```diff
@@ -0,0 +1,53 @@
+-- Find out how many tasks are in the task table
+SELECT COUNT(*)
```

Comment:
COUNT(*) as task_amount
_Pointer_
Provide a column name for columns you have created